matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/permalinks/PermalinkParser.kt

```diff
@@ -53,7 +53,12 @@ object PermalinkParser {
                 .filter { it.isNotEmpty() }
                 .take(2)
 
-        val identifier = params.getOrNull(0)
+        // the base domain permalinks don't have the mxid in the first param but in the second after /user/mxid
```

Comment:
Can you provide an example of such URL? Maybe in the doc of this method.

Reply:
added an example. Thx for reviewing